mix.exs

```diff
@@ -85,12 +85,18 @@ defmodule Appsignal.Mixfile do
         _ -> ">= 1.3.0"
       end
 
+    phoenix_version =
+      case Version.compare(System.version(), "1.4.0") do
```

Comment:
can we assign `System.version()` to a variable so we don't have to call it twice? Here and on line 83.

Reply:
Oh I see it was actually part of #496